zksync_sdk/serializers.py

```diff
@@ -71,6 +75,33 @@ def integer_to_float(integer: int, exp_bits: int, mantissa_bits: int, exp_base:
     return result
 
 
+def integer_to_float_up(integer: int, exp_bits: int, mantissa_bits: int, exp_base) -> List[int]:
+    max_exponent_power = pow(2, exp_bits) - 1
+    max_exponent = pow(exp_base, max_exponent_power)
+    max_mantissa = pow(2, mantissa_bits) - 1
```

Comment:
In `integer_to_float` function `**` is used instead of `pow()`. I think it is better to use the same way in both methods for consistency